tests/Integration/RemoteAPI/content-suggestions/BaseContentSuggestionsAPITest.php

```diff
@@ -58,4 +58,60 @@ public function test_api_headers(): void {
 		self::assertArrayHasKey( 'X-APIKEY-SECRET', $headers );
 		self::assertEquals( 'my-secret', $headers['X-APIKEY-SECRET'] );
 	}
+
+	/**
+	 * Verifies that the truncate function is properly truncated long content on the body array.
+	 *
```

Comment:
Add a `@since` here, and check for coverage in case more functions need to be added as `@covers` or `@uses`.

Reply:
Since only `truncate_array_content` is being called, that's the only requirement in `@covers`. I have validated this with the coverage command.

Reply:
Thank you!